tests/storage/lvm_test.py

```diff
@@ -1023,6 +1023,42 @@ def test_vg_create_multiple_devices(tmp_storage):
         assert pv.vg_name == ""
 
 
+def test_vg_get_non_existing_vg(fake_devices):
+    fake_runner = FakeRunner()
+    lc = lvm.LVMCache(fake_runner)
+
+    vg_name = "existing-vg-name"
+    non_vg_name = "non-existing-vg-name"
+
+    # Create fake devices.
+    fake_pv = make_pv(pv_name="pv", vg_name=vg_name)
+    fake_vg = make_vg(pvs=[fake_pv.name], vg_name=vg_name)
+
+    # Assign fake PV, VG to cache.
+    lc._pvs = {fake_pv.name: fake_pv}
+    lc._vgs = {fake_vg.name: fake_vg}
+
+    # VG is returned with valid VG name.
+    vg = lc.getVg(vg_name)
+    assert vg.name == vg_name
+
+    # Non-existing VG name raises an exception.
+    with pytest.raises(se.VolumeGroupDoesNotExist) as e:
+        lc.getVg(non_vg_name)
+    # The name of the non-found VG must appear in the exception.
+    assert non_vg_name in e.value.value
```

Comment:
Because the error message may change in the future, be reformulated a bit, and it would be a bit of a burden having to update the check in all asserts that look into the error message.

The relevant part for me was that the vg_name that caused the exception to rise appears in the message, and so this check.

But I'm ok with changing so that the whole error message is checked with `==` if you think that adds verification value.